src/lib/store.js

```diff
@@ -49,7 +49,8 @@ const initialState = {
   // cookie policy.
   // We automatically accept cookies in dev and test environments so the cookie
   // banner doesn't interfere with tests.
-  userAcceptsCookies: !isProd,
+  // cookiePreference: isProd ? null : 'accepts',
```

Comment:
I think we need the isProd check for visual test to be consistent.

Reply:
Oops, had commented this out to test and forgot to add it back. Good catch! Fixed in https://github.com/GoogleChrome/web.dev/pull/9168/commits/c7174162afacc5198aa1345b2375775b03637fef.